File: 250407/500 근처의 수/near-500.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int arr[] = new int[10];

        for(int i = 0; i < 10; i++) {
            arr[i] = sc.nextInt();
        }
        int max = Integer.MIN_VALUE;
        int standard = 500;
        int min = Integer.MAX_VALUE;
        for(int i = 0 ; i < 10; i++) {
            if(arr[i] > max && arr[i] < standard) {
                max = arr[i];
            }
            if(arr[i] < min && arr[i] > standard) {
                min = arr[i];
            }

        }
        System.out.printf("%d %d", max, min);
        sc.close();
    }
}

File: 250407/500 근처의 수/near-500.cpp
#include <iostream>
#include <climits>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int arr[10] = {0};
    for(int i = 0; i < 10; i++) {
        cin >> arr[i];
    }
    int max = INT_MIN;
    int standard = 500;
    int min = INT_MAX;
    for(int i = 0; i < 10; i++) {
        if(arr[i] > max && arr[i] < standard) {
            max = arr[i];
        }
        if(arr[i] < min && arr[i] > standard) {
            min = arr[i];
        }
    }
    cout << max << " " << min;
    return 0;
}